presto-orc/src/test/java/com/facebook/presto/orc/TestLongDictionaryProvider.java

```diff
@@ -287,8 +289,12 @@ private Set<Integer> getSharedColumnIds(Map<NodeId, long[]> dictionaryStreams)
 
     private StreamDescriptor createFlatStreamDescriptor(StreamId streamId)
     {
-        return new StreamDescriptor("test_dictionary_stream", streamId.getColumn(),
-                String.format("field_%d", streamId.getColumn()), LONG_TYPE, DUMMY_ORC_DATA_SOURCE, ImmutableList.of(), streamId.getSequence());
+        Map<Integer, StreamProperty> streamProperties = ImmutableMap.of(
+                streamId.getColumn(),
+                new StreamProperty("test_dictionary_stream", LONG_TYPE, String.format("field_%d", streamId.getColumn()), ImmutableList.of()));
```

Comment:
Nit: can you please replace `String.format("field_%d", streamId.getColumn())` with a simple concatenation?